walkthroughs/1-ionic-showcase/walkthrough.adoc

```diff
@@ -212,57 +132,52 @@ Verify that you followed each step in the procedure above.  If you are still hav
 ****
 
 [time=15]
-== Protect the mobile showcase app using the Identity Management service
-
-In this task, you use the Identity Management service to protect both the client (PWA) and the data sync server.
-
-=== Add Identity Management service to the client app (PWA) and create a realm for users
+== Protect the GraphQL showcase app using the Red Hat SSO
 
-. Navigate to the link:{mdc-url}[Mobile Developer Console, window="_blank"].
-. Click on the *showcase-app* card to display the configuration screen.
-. Click the *Mobile Services* tab.
-. Click *Bind to App* for *Identity Management* in the *Unbound Services* section and then click the *next* button.
-.. When prompted for an *Admin User Name* enter:
-+
-----
-admin
-----
-.. When prompted for an *Admin User Password* enter:
-+
-----
-admin
-----
-. Complete the wizard.
-This process also creates a new realm in the {customer-sso-name}.
+In this task, you use the Red Hat SSO to protect both the client (PWA) and the data sync server.
 
-. Navigate to the *Configuration* tab of the *mobile app* in the Mobile Developer Console.
-
-. Copy and save the `mobile-services.json` file, you will require this file in a later task.
-
-[type=verification]
-****
-Is *Identity Management* row listed the `Bound Services` section for *showcase-app*?
-****
+=== Connect SSO to the client app
 
-[type=verificationFail]
-****
-Verify that you followed each step in the procedure above.  If you are still having issues, contact your administrator.
-****
+. Make sure you have access to a realm in the user-facing SSO instance. If you don't have access, you can ask your customer-admin user to create one for you. 
+. In new realm we going to create two new clients - one for server side and one for the client side application.
+. Open URL obtained from adminstrator
+. Login to keycloak (you will be asked to provide additional details like email and name)
+. When completing login process you should see the main keycloak realm page
+. In the right menu choose *Clients* under *Configure* in the menu.
```

Comment:
Is this in the on the right side? Is it not on the left side?

Also you're saying 'in the menu choose x in the menu'

Maybe it could be something like 

```suggestion
. In the menu on the right side, choose *Configure* > *Clients*.
```